.changeset/big-bugs-kick.md

```diff
@@ -0,0 +1,7 @@
+---
+'@keystonejs/app-admin-ui': patch
+'@keystonejs/fields': patch
```

Comment:
```suggestion
'@keystonejs/fields': major
```

Because we're adding a new required field (`createItemModal`) to `Relationship` field, this is technically a breaking change.

Reply:
this does not change anything usable outside of the admin-ui yet, should this still be `major`, I suggest `minor`

Reply:
Jess is right, it's an API change and because it's required it's technically breaking. You also right that this is pretty deep in the bowels of a non-public part of the API. I'd be happy with anything above pach.

Reply:
changed to minor